src/Storages/ExternalStream/Kafka/WriteBufferFromKafka.cpp

```diff
@@ -0,0 +1,80 @@
+#include "WriteBufferFromKafka.h"
+#include <cmath>
+#include <utility>
```

Comment:
`<utility>` for ?

Reply:
The vector  was added accidentially by the language server.